### ISSUE TITLE
Deno panics if target is not specified, when run `deno doc` 

### ISSUE THREAD

**Issue by m1sk9** [denoland/deno#21116]
Deno panics if no target argument is specified when `deno doc` is executed.

```
Platform: macos aarch64
Version: 1.38.0
Args: ["deno", "doc", "--html", "--name=pulsate"]

thread 'main' panicked at /Users/ib/.cargo/registry/src/index.crates.io-6f17d22bba15001f/deno_doc-0.72.2/src/html/mod.rs:638:3:
assertion failed: !paths.is_empty()
```

**Backtrace:**

```
stack backtrace:
   0:        0x1056fdeb8 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h6d4268b2ed62fb94
   1:        0x104c93790 - core::fmt::write::h5d55d44549819258
   2:        0x1056d1ac4 - std::io::Write::write_fmt::hc515897f91abd6cf
   3:        0x1057025fc - std::sys_common::backtrace::print::h2c300c1ebedfc73c
   4:        0x105702254 - std::panicking::default_hook::{{closure}}::h0aa9be5c44269370
   5:        0x105701f20 - std::panicking::default_hook::h2c0ef097934ee9e6
   6:        0x104bd03a0 - deno::setup_panic_hook::{{closure}}::h2ab29d316013a1d9
   7:        0x105702ee8 - std::panicking::rust_panic_with_hook::h84c8637cb6e56008
   8:        0x105702ca8 - std::panicking::begin_panic_handler::{{closure}}::h25482adda06c7b7f
   9:        0x105702c3c - std::sys_common::backtrace::__rust_end_short_backtrace::h0c6f3beb22324a29
  10:        0x105702c30 - _rust_begin_unwind
  11:        0x104c91a3c - core::panicking::panic_fmt::h9072a0246ecafd14
  12:        0x104c934a8 - core::panicking::panic::hcabaf8436b9c6302
  13:        0x104e6b578 - deno_doc::html::generate::h9a072304c9e13f16
  14:        0x104b81cd8 - deno::tools::doc::generate_docs_directory::{{closure}}::h275c3e403038c35e
  15:        0x104b78e30 - deno::tools::doc::doc::{{closure}}::h63083cc3d7e6f5b1
  16:        0x104b77980 - deno::spawn_subcommand::{{closure}}::ha1a3f8ef98abfcec
  17:        0x104b9cc18 - <deno_unsync::task::MaskFutureAsSend<F> as core::future::future::Future>::poll::hd7d22c2d80b3fd89
  18:        0x104b9cf50 - tokio::runtime::task::raw::poll::h537b84e571da0497
  19:        0x1048ac628 - deno::main::hbbd064f0d9427d55
  20:        0x10489e9c8 - std::sys_common::backtrace::__rust_begin_short_backtrace::h618b86861eac80e4
  21:        0x1048ab06c - _main
```

**Comment by bartlomieju** [comment]
This has been already fixed in https://github.com/denoland/deno/pull/21072 and will be released in Deno v1.38.1 today.

**Comment by m1sk9** [comment]
Thanks!
